packages/jest-config/src/normalize.js

```diff
@@ -185,9 +185,12 @@ function normalize(config, argv) {
   if (config.scriptPreprocessor) {
     config.scriptPreprocessor =
       _replaceRootDirTags(config.rootDir, config.scriptPreprocessor);
-    if (config.scriptPreprocessor.includes(
-      constants.NODE_MODULES + 'babel-jest',
-    )) {
+    if (
+      config.scriptPreprocessor.includes(
+        constants.NODE_MODULES + 'babel-jest',
+      )
+      || config.scriptPreprocessor.includes('packages/babel-jest')
```

Comment:
why do we need this? I'm hoping only for Jest's tests themselves?

Since we talked about adding a flag to the preprocessor today. Instead of this shitty detection based on a path, can we instead make it so the preprocessor exports a flag to say that it is babel (`"isBabel: true"`, to keep it generic)? That way we can just require it here and see if it is babel jest.

(you can do this in a follow-up, but I wanted to bring it up here as I was never happy with this detection method when I introduced it).